x-pack/plugins/maps/public/components/global_filter_checkbox.tsx

```diff
@@ -21,13 +22,21 @@ export function GlobalFilterCheckbox({ applyGlobalQuery, label, setApplyGlobalQu
 
   return (
     <EuiFormRow display="columnCompressedSwitch">
-      <EuiSwitch
-        label={label}
-        checked={applyGlobalQuery}
-        onChange={onApplyGlobalQueryChange}
-        data-test-subj="mapLayerPanelApplyGlobalQueryCheckbox"
-        compressed
-      />
+      <EuiToolTip
+        position="top"
+        content={i18n.translate('xpack.maps.filterEditor.applyGlobalFilterHelp', {
+          defaultMessage:
+            'When enabled, the query from the global search bar is applied to this layer"',
```

Comment:
GlobalFilterCheckbox is used in two places, Layer filtering configuration and Term join configuration. The message needs to be usable in both places and as such should not contain the phrase "this layer".

How about just "When enabled, results narrowed by global search"